webpack/published-converted-components.js

```diff
@@ -0,0 +1,19 @@
+const path = require("path");
+const publishedComponents = {
+  Button: "components/Button/Button",
+  AlertBannerButton: "components/AlertBanner/AlertBannerButton",
+  ToastButton: "components/Toast/ToastButton/ToastButton"
```

Comment:
These components still appear in the `published-components` file, is this intentional?

Reply:
I saw what you wrote in the other note about this problem. I will change it